NitroxPatcher/Patches/Dynamic/GoTo_Patch.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+using HarmonyLib;
+using NitroxModel.Helper;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class GoTo_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((GotoConsoleCommand gotoConsoleCommand) => gotoConsoleCommand.GotoPosition(default, default));
+
+    public static bool Prefix(Vector3 position, bool gotoImmediate = false)
+    {
+        Vehicle currentMountedVehicle = Player.main.currentMountedVehicle;
+        if (currentMountedVehicle == null)
```

Comment:
Same here with checking against null 
```suggestion
        if (!currentMountedVehicle)
```